OtripleS.Web.Api.Tests.Unit/Services/StudentGuardianServiceTests/StudentGuardianServiceTests.Validations.RetrieveById.cs

```diff
@@ -88,5 +88,46 @@ public async Task ShouldThrowValidatonExceptionOnRetrieveWhenGuardianIdIsInvalid
             this.loggingBrokerMock.VerifyNoOtherCalls();
             this.dateTimeBrokerMock.VerifyNoOtherCalls();
         }
+
+        [Fact]
+        public async Task ShouldThrowValidationExceptionOnRetrieveWhenStorageStudentGuardianIsInvalidAndLogItAsync()
+        {
+            // given
+            DateTimeOffset randomDateTime = GetRandomDateTime();
+            StudentGuardian randomStudentGuardian = CreateRandomStudentGuardian(randomDateTime);
+            Guid inputSemasterCourseId = randomStudentGuardian.GuardianId;
```

Comment:
@AgentEnder maybe it is better to have `inputGuardianId` here

Reply:
Ah, my bad. I used the StudentSemesterCourse tests as a template and missed this due to Semaster vs Semester.